src/screens/AffiliationViews/DeploymentView/MatrixView/Matrix.tsx

```diff
@@ -21,15 +21,18 @@ export const Matrix: React.FC<IMatrixProps> = ({
     return <Spinner />;
   }
 
-  const environments = deployments.reduce(
-    (acc, app) => {
-      if (acc.indexOf(app.environment) === -1) {
-        return acc.concat(app.environment);
-      }
-      return acc;
-    },
-    [' ']
-  );
+  const appCountForEnv = deployments.reduce((prev, cur) => {
+    prev[cur.environment] = (prev[cur.environment] || 0) + 1;
+    return prev;
+  }, {});
+
+  const environments = Object.keys(appCountForEnv)
+    .sort()
+    .sort((a, b) => {
+      return appCountForEnv[b] - appCountForEnv[a];
```

Comment:
Usikker på om dere pleier å ha en konvesjon for funksjons blokker, men her hadde jeg fjernet brackets og return slik at det ble .sort((a, b) => appCountForEnv[b] - appCountForEnv[a])

Reply:
Vi har ikke en spesifikk konvesjon for funksjons blokker. Enig i at det er bedre å droppe dem i dette caset.